diesel_cli/Cargo.toml

```diff
@@ -23,6 +23,7 @@ infer_schema_internals = { version = "1.1.0" }
 clippy = { optional = true, version = "=0.0.185" }
 migrations_internals = { version = "1.1.0" }
 url = { version = "1.4.0", optional = true }
+barrel = { git = "https://github.com/spacekookie/barrel", optional = true, features = ["diesel"] }
```

Comment:
Are you planning on releasing before this is merged? I'm not comfortable putting a git dependency here.